presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -2374,29 +2345,27 @@ velox::core::PlanFragment VeloxBatchQueryPlanConverter::toVeloxQueryPlan(
   auto partitionedOutputNode =
       std::dynamic_pointer_cast<const core::PartitionedOutputNode>(
           planFragment.planNode);
-  VELOX_CHECK(
-      partitionedOutputNode != nullptr, "PartitionedOutputNode is required");
+  VELOX_CHECK_NOT_NULL(
+      partitionedOutputNode, "PartitionedOutputNode is required");
   if (partitionedOutputNode->isBroadcast()) {
     VELOX_UNSUPPORTED(
-        "Broadcast partitioned output node in batch is currently not "
+        "Broadcast partitioned output node in batch is not supported.");
+  }
+
+  if (partitionedOutputNode->isReplicateNullsAndAny()) {
```

Comment:
TIL: replicate-nulls-and-any is not supported in batch.
Can you please explain what makes batch not support replicate-nulls-and-any, while interactive mode has the support.

Reply:
@vermapratyush We just didn't implement it. This would go into PartitionAndSerialize operator.

See https://facebookincubator.github.io/velox/develop/anti-join.html